components/addressmanager/src/lib.rs

```diff
@@ -48,19 +36,19 @@ impl AddressManager {
         }
 
         // We mark `connection_failed_count` as 0 only after first success
-        self.not_banned_address_store.set(address, ConnectionFailureCount(1));
+        self.not_banned_address_store.set(address, 1);
```

Comment:
I would switch `not_banned` to something like -> `nonbanned`/`allowed`/`legitimate ` in all struct/mod/var names